src/Payum/Be2Bill/Action/CaptureOffsiteNullAction.php

```diff
@@ -39,10 +40,11 @@ public function execute($request)
 
         $this->gateway->execute($getToken = new GetToken($extraData['capture_token']));
 
-        $url = Url::createFromUrl($getToken->getToken()->getTargetUrl());
-        $url->setQuery($httpRequest->query);
+        $uri = HttpUri::createFromString($getToken->getToken()->getTargetUrl());
+        $uri = $uri->withQuery(Query::createFromArray($httpRequest->query)->__toString());
```

Comment:
Your are right. I was thinking about that. They use __toString() in their examples thus I decided to keep it, but I did not change  (string) $uri.